clusterloader2/pkg/measurement/common/kcm_metrics.go

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
I see component metrics related files are all placed under `https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/pkg/measurement/common/metrics`.  If there are two parts to this file - collecting metrics and measurement logic, can you move the former to that location and rename this file to explain what the measurement we are making is? Just like the others - https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/pkg/measurement/common